src/global.css

```diff
@@ -1,53 +1,41 @@
-html {
-  font-size: 1.125rem;
-  line-height: 1.6;
-  font-weight: 400;
-  font-family: sans-serif;
-  box-sizing: border-box;
-  scroll-behavior: smooth;
-  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
-}
-
-*,
-*::before,
-*::after {
-  box-sizing: inherit;
+.App {
+  width: 100vw;
+  height: 100vh;
+  text-align: center;
 }
```

Comment:
I recommend using the styled-components approach for adding CSS to the components. I have a comment with more details on the recommended approach